src/AasxServerDB/PageDB.cs

```diff
@@ -0,0 +1,130 @@
+﻿namespace AasxServerDB
+{
+    public class PageDB

```

Comment:
Improve naming of the class and method and apply the suggestiosn from the other classes as well